### ISSUE TITLE
Generating images on 8 GPUs

### ISSUE THREAD

**Issue by sudanl** [starriver030515/SynthVLM#2]
Hi! Awesome work! 

I would like to ask how to synthesize images on 8 GPUs (as mentioned in the paper)? Are they generated separately on each of the 8 GPUs?

**Comment by starriver030515** [comment]
Yes, the existing diffusion models only supports single GPU, but you can write a bash script to run it in parallel across 8 GPUs.
